src/main/scala/com/fulcrumgenomics/sopt/Sopt.scala

```diff
@@ -56,7 +58,25 @@ object Sopt {
                          hidden: Boolean,
                          description: String,
                          args: Seq[Arg]
-                        ) extends MarkDownDescription
+                        ) extends MarkDownDescription {
+    /** Returns a command line string representation of the arguments.  If user-specified values are set on the args,
+      * it will use those, otherwise the defaults if present, otherwise it the type of the argument.
+      *
+      * @param withDefaults true to include args with defaults, otherwise those with user-set values.
```

Comment:
Not sure if the doc is wrong or I'm misunderstanding.  I think you _always_ use the user set values, and if `withDefaults` is true, you also include options that have default values but no user set value?

Reply:
I think you're right to be confused. If I rename this `includeArgsWithDefaults`:

* If `false`, then only include arguments who have values set by the user.
* If `true`, also include arguments with default values.